skins/laika/src/components/pages/PrivacyProtection.vue

```diff
@@ -1,37 +1,77 @@
-            <div class="privacy_selection_box">
-                <div class="wrap-field choice-in">
-                    <div class="wrap-input">
-                        <input name="matomo_choice" id="tracking-opt-in" value="0" type="radio">
-                        <label for="tracking-opt-in"><span>{{ $t('privacy_optout_tracking_permit') }}</span></label>
-                    </div>
-                </div>
-                <div class="wrap-field choice-out">
-                    <div class="wrap-input">
-                        <input name="matomo_choice" id="tracking-opt-out" value="1" type="radio">
-                        <label for="tracking-opt-out"><span>{{ $t('privacy_optout_tracking_deny') }}</span></label>
-                    </div>
-                </div>
-            </div>
-            <div class="privacy_explanation">
-                <p>
-                    {{ $t('privacy_optout_tracking_state') }}
-                </p>
-            </div>
-        </div>
+            <b-radio id="tracking-opt-in"
+                    name="matomo_choice"
+                    :native-value="0"
+                    v-model="optOut"
+                    @input="changeTracking">
+                {{ $t('privacy_optout_tracking_permit') }}
+            </b-radio>
+            <b-radio id="tracking-opt-out"
+                    name="matomo_choice"
+                    :native-value="1"
+                    v-model="optOut"
+                    @input="changeTracking">
+                {{ $t('privacy_optout_tracking_deny') }}
+            </b-radio>
+            <p v-if="optOut === 0" class="has-text-dark-lighter has-margin-top-18">{{ $t( 'privacy_optout_tracking_state' ) }}</p>
+            <p v-else class="has-text-dark-lighter has-margin-top-18" v-html="$t( 'privacy_optout_tracking_state_no' )"></p>
     </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
+import jsonp from 'jsonp';
+import { TRACKING_URL } from '@/trackingUrl';
 
 export default Vue.extend( {
 	name: 'PrivacyProtection',
+	data: function () {
+		return {
+			optOut: 0,
+		};
+    },
+    beforeMount: function() {
+       this.getInitialTrackingState();
+    },
+	methods: {
+		changeTracking: function (): void {
+			if ( this.$data.optOut === 0 ) {
+				jsonp( TRACKING_URL + 'index.php?module=API&method=AjaxOptOut.doTrack&format=json',
+					undefined,
+					function () {}
+				);
+			} else {
+				jsonp( TRACKING_URL + 'index.php?module=API&method=AjaxOptOut.doIgnore&format=json',
+					undefined,
+					function () {}
+				);
+			}
+        },
+        getInitialTrackingState: function(): void {
+            jsonp( TRACKING_URL + 'index.php?module=API&method=AjaxOptOut.isTracked&format=json',
+					undefined,
+					( error, data ) => {
+                        this.$data.optOut = data.value ? 0 : 1;
+                    }
+				);
+        },
+	},
 } );
 </script>
+<style lang="scss" scoped>
+@import "../../scss/custom.scss";
+	.b-radio.radio {
+		&+.radio{
```

Comment:
Not sure why the linter is not complaining but this file has mixed spaces and tabs in the `script` and `style` sections and also some formatting issues like the missing space before the bracket here.

Reply:
Yeah not sure why the linter doesn't complain. That's why I randomly fix mixed tabs and spaces in some files here and there.